bodo/pandas/utils.py

```diff
@@ -318,6 +318,19 @@ def __init__(self, plan_class, *args, **kwargs):
         self.output_func = None  # filled in by wrap_plan
         self.out_schema = None  # filled in by wrap_plan
 
+    def __str__(self):
+        out = f"{self.plan_class}: \n"
+        for arg in self.args:
+            if isinstance(arg, pd.DataFrame):
+                out += f"  {arg.columns.tolist()}\n"
+                continue
+            out += f"  {arg}\n"
+        for k, v in self.kwargs.items():
+            out += f"  {k}: {v}\n"
+        return out
+
+    __repr__ = __str__
```

Comment:
This isn't a __repr__ is it because you can't reproduce the object from the information output?

Reply:
This isn't a proper `__repr__` but added it for convenience to print objects more easily inside gdb.